dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLiteConnector.java

```diff
@@ -30,18 +31,22 @@
 import com.google.edwmigration.dumper.application.dumper.task.Summary;
 import com.google.edwmigration.dumper.application.dumper.task.Task;
 import com.google.edwmigration.dumper.application.dumper.utils.ArchiveNameUtil;
-import com.google.edwmigration.dumper.plugin.lib.dumper.spi.SnowflakeMetadataDumpFormat;
+import com.google.edwmigration.dumper.plugin.lib.dumper.spi.SnowflakeMetadataDumpFormat.DatabasesFormat;
+import com.google.edwmigration.dumper.plugin.lib.dumper.spi.SnowflakeMetadataDumpFormat.SchemataFormat;
+import com.google.edwmigration.dumper.plugin.lib.dumper.spi.SnowflakeMetadataDumpFormat.TablesFormat;
 import java.time.Clock;
 import java.util.List;
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
 
 @AutoService(Connector.class)
-public final class SnowflakeLiteConnector extends AbstractSnowflakeConnector
-    implements SnowflakeMetadataDumpFormat {
+@ParametersAreNonnullByDefault
+public final class SnowflakeLiteConnector extends AbstractSnowflakeConnector {
 
   private static final String NAME = "snowflake-lite";
```

Comment:
nit: `CONNECTOR_NAME`

Reply:
This is intended to match the `name` super-constructor parameter for the `getName` method in `AbstractConnector` and another parameter in `ArchiveNameUtil` which is also called `name`. It's not used in anywhere else, so the current version is more consistent.